src/test/atlas_connectivity.rs

```diff
@@ -49,3 +49,15 @@ async fn atlas_repl_set_srv() {
     )
     .await;
 }
+
+#[cfg_attr(feature = "tokio-runtime", tokio::test)]
+#[cfg_attr(feature = "async-std-runtime", async_std::test)]
+async fn atlas_serverless() {
+    run_test("MONGO_ATLAS_SERVERLESS_URI", None).await;
+}
+
+#[cfg_attr(feature = "tokio-runtime", tokio::test)]
+#[cfg_attr(feature = "async-std-runtime", async_std::test)]
+async fn atlas_serverless_srv() {
+    run_test("MONGO_ATLAS_SERVERLESS_URI_SRV", None).await;
```

Comment:
For this one, can we add the explicit resolver config like in the regular Atlas SRV test? This will speed the test up greatly on Windows.

Reply:
Done, although I admit I'm confused how adding a second test run will speed it up.

Reply:
Oh sorry, I meant replacing this run with one that uses the custom resolver, not adding a second one.

Reply:
Ah, yes, derp.  Updated.